### PR TITLE
prompt for namespace after uploading a license

### PR DESCRIPTION
**Pull request by GraysonNull**

### CODE REVIEW

File: api/src/kots_app/kots_app_store.ts [hunk 1]
```diff
@@ -67,6 +67,16 @@ export class KotsAppStore {
     await this.pool.query(q, v);
   }
 
+  async updateNamespace(id: string, namespace: string): Promise<void> {
+    const q = `update app set namespace = $1 where id = $2`;
```
Comment (divolgin) [inline api/src/kots_app/kots_app_store.ts]: This column is used for registry namespace, not app namespace

Comment (marccampbell) [reply api/src/kots_app/kots_app_store.ts]: @divolgin do we currently have an app_downstream namespace column?

Comment (GraysonNull) [reply api/src/kots_app/kots_app_store.ts]: Ah, do we need to add a new column for app namespace to this table or is that managed somewhere else?

**Comment by marccampbell** [comment]
There a bigger project that's WIP to include this. Closing this PR

**Pull request closed without merging**